containerfiles/rocky-latest-doca-ofed

```diff
@@ -0,0 +1,60 @@
+# Based on https://github.com/openstack/diskimage-builder/blob/master/diskimage_builder/elements/rocky-container/containerfiles/9
+
+FROM quay.io/rockylinux/rockylinux:9
+
+# Workaround for containerized kernel tools
+RUN echo "9.6" > /etc/dnf/vars/releasefullver && \
+# Can't update packages in DIB once OFED is installed as it provides some "older" packages
+RUN dnf update -y \
+  ; dnf -y install chkconfig
+
+RUN dnf group install -y 'Minimal Install' --allowerasing && \
+    dnf install -y findutils util-linux sudo python3 NetworkManager \
+    NetworkManager-config-server curl libcurl --allowerasing
+
+RUN systemctl unmask console-getty.service dev-hugepages.mount \
+    getty.target sys-fs-fuse-connections.mount systemd-logind.service \
+    systemd-remount-fs.service
+
+# /etc/machine-id needs to be populated for /bin/kernel-install to
+# correctly copy kernels into /boot.  We will clear this out in the
+# final image.
+RUN systemd-machine-id-setup
+
+# Install base build dependencies required by DOCA OFED
+RUN dnf config-manager --set-enabled appstream crb && \
```

Comment:
Can we move these into a DIB element? Leaving all packages in the image is a security anti-pattern - at least let's remove them as a final step?